charts/fleet-crd/104.0.0+up0.10.0-rc.3/README.md

```diff
@@ -0,0 +1,5 @@
+# Fleet CRD Helm Chart
```

Comment:
I'm surprised not to see actual CRDs in this chart (from [here](https://github.com/rancher/fleet/tree/master/charts/fleet-crd/templates)) :thinking:

Reply:
Since we switched to codegen, https://github.com/rancher/fleet/blob/master/scripts/package-helm#L39 is outdated and would need to use 
`bash ./cmd/codegen/hack/generate_and_sort_crds.sh ./build/charts/fleet-crd/templates/crds.yaml`.


However, I don't think the release scripts should generate CRDs and such. Let's just release what's checked into the repo. That's what we tested.